examples/basic_mysql_connection/index.js

```diff
@@ -2,30 +2,33 @@ var fs = require('fs');
 var mysql = require('mysql');
 
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
-
+  
 var connection = mysql.createConnection({
-	 host: config.host,
-	user: config.user,
-	password: config.password,
-	port: config.port,
-  database: config.database
+ 	  host: config.host,
+  	user: config.user,
+  	password: config.password,
+  	port: config.port,
+    database: config.database
 });
-
-connection.connect();
-const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
-					'SELECT * FROM todos WHERE Done',
-					'SELECT * FROM todos WHERE !Done',	
-					'SELECT * FROM todos GROUP BY Id DESC',
-					'SELECT * FROM todos  ORDER BY id DESC LIMIT 1',
-					'SELECT * FROM todos WHERE Name like "%databases%"',
-
-				  ]
-let sql = sqlStatments.map(function(value) {
-  connection.query(value, function(error, results, fields) {
-    console.log(results);
-	console.log("=========================================================");
-  });
-
-
-})
-connection.end();
\ No newline at end of file
+  
+   
+const sqlStatments=[
+        `INSERT INTO todos (Done, Name, StatusId, Due) VALUES (1, 'Review SCSS', 1,'2017-08-17 00:00:00')`,
+        `UPDATE todos SET Name='HTML5' WHERE Id=25`,
+        `UPDATE todos SET StatusId=(select Id FROM statuses WHERE Name='Completed') WHERE Id=27`,
+        `DELETE FROM todos WHERE Id=27`,
+        `DELETE FROM todos WHERE Id=28`   
```

Comment:
+1 to Jalal's comment. You should use `mysql.escape()` anywhere that the data would be coming from untrusted user input. Which values in these statements would probably be coming from user input?